Doc/c-api/structures.rst

```diff
@@ -472,6 +491,24 @@ definition with the same method name.
    .. versionadded:: 3.9
 
 
+.. c:var:: PyTypeObject PyCFunction_Type
+
+   The type object corresponding to Python C function objects. This is
+   available as :class:`types.BuiltinFunctionType` in the Python layer.
+
+
+.. c:function:: int PyCFunction_Check(PyObject *f)
```

Comment:
Keep a consistent name for the parameter. For `_Check` functions, I would prefer having `op` everywhere.